internal/api/ping.go

```diff
@@ -0,0 +1,26 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+
+	logger "github.com/sirupsen/logrus"
+)
+
+// PingResponse - a basic struct to represent the JSON response for /ping
+type PingResponse struct {
+	Message string `json:"message"`
```

Comment:
use generic response struct instead of creating new one for each API, that too for just message string.
@SamnitPatil9882 @SharyuMarwadi